valheim_plus.cfg

```diff
@@ -232,6 +238,22 @@ dataRate=60
 ; The interval in seconds that the game auto saves at (client only)
 autoSaveInterval=1200
 
+[Sleep]
+enabled=false
+; Change false to true to enable this section
```

Comment:
For convention, let's move the comments above the property declarations

Reply:
We should declare that somewhere since they were below them in previous iterations of the file.

Reply:
Regardless, once I'm able to it's an easy change, I can move them.